BanyanONNXRuntime/res/pf_dispatch_table.toml

```diff
@@ -0,0 +1,11 @@
+
+# TODO: THIS NEEDS TO BE IMPLEMENTED
+
+
+[splits.ReadBlockONNX]
+expected = [ "key" ]
+partition_memory_usage = "part"
+scale_by = 2
+owns_partitions = false
+required = { name = "Distributing", distribution = "blocked", id = "!" }
+locations = [ {name = "Remote", format = "onnx"}, { name = "Disk", datatype = "ONNX" } ]
```

Comment:
Comments:
- `ReadONNX`
- Maybe call it `ONNXSession` (is that what it's called?) datatype. Also make sure to pass in `datatype` into the `Future` constructor for any future that stores an ONNXSession.
- `scale_by` should just be 1 and `partition_memory_usage` should be "all" (or "whole", whatever it is for replicating PFs)
- No expected PT parameters and the required PT parameter should be the same as it is for replicated computation (like `CopyFrom`)